packages/glslang/build.sh

```diff
@@ -0,0 +1,31 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/KhronosGroup/glslang
+TERMUX_PKG_DESCRIPTION="OpenGL and OpenGL ES shader front end and validator"
+TERMUX_PKG_LICENSE="BSD"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=11.13.0
+TERMUX_PKG_SRCURL=https://github.com/KhronosGroup/glslang/archive/$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SHA256=592c98aeb03b3e81597ddaf83633c4e63068d14b18a766fd11033bad73127162
+TERMUX_PKG_DEPENDS="python"
```

Comment:
Is `python` needed?

Reply:
Specified as *build* dependency. I don't believe it is needed on run-time.

Reply:
BTW, from time to time I see PRs in which `ndk-sysroot` is specified as dependency, which is incorrect.

Reply:
Okay, python is not needed to compile the package, but then I will indicate it in the recommendation

Reply:
Not even needed in the recommendation.

Reply:
Very high risks that problems with glslang may occur.